operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Entries should be ordered as follows:
 
 Entries should include a reference to the Pull Request that introduced the change.
 
+## 2.2.0-beta.1
```

Comment:
I don't have much of a preference, the tooling that I reused works this way and for me it's ok to see separate changes in separate chart releases. Let's ask @Logiraptor  @56quarters @dimitarvdimitrov  as well.

Reply:
nothing against granular releases, but I don't see great use in them. My weak preference is for weekly/bi-weekly beta releases, ideally automated.

Reply:
A weekly chart release could also be rolled into the internal weekly rollouts for our dev environment. We don't have any automatic helm chart bump there yet.

Reply:
Also during the 2.1 release it was really annoying to constantly fix conflicts in the CHANGELOG, Chart.yaml, and README since the version bump has to be reflected in all three.

Reply:
updated